install/ansible/install.sh

```diff
@@ -86,6 +87,22 @@ fi
 
 ansible_path=./ansible
 env_file=install/ansible/env.json
+# Verify ansible can reach all hosts
+
+echo "Verifying ansible reachability"
+ansible all $ans_opts -i $host_inventory -m setup -a 'filter=ansible_distribution*' >& $inventory_log
+egrep 'FAIL|UNREACHABLE' $inventory_log >& /dev/null
+if [ $? -eq 0 ]; then
+   echo "WARNING"
+   echo "WARNING Some of the hosts are not accessible via passwordless SSH"
+   echo "WARNING"
+   echo "`egrep 'FAIL|UNREACHABLE' $inventory_log`"
+   echo " "
+   echo "This means either the host is unreachable or passwordless SSH is not"
+   echo "set up for it. It is RECOMMENDED that you resolve this before proceeding."
```

Comment:
Is it really _recommended_ if the script exits with a 1 right afterwards instead of continuing? :D